soh/src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6668,7 +6668,14 @@ s32 func_8083F0C8(Player* this, PlayState* play, u32 arg2) {
                 this->actor.world.pos.z = tempZ + (wallDistance * wallPolyNormZ);
                 func_80832224(this);
                 this->actor.prevPos = this->actor.world.pos;
-                func_80832264(play, this, &gPlayerAnim_link_child_tunnel_start);
+                if (CVarGetInteger("gCrawlSpeed", 0) < 2) {
```

Comment:
Every Cvar get here in this file also needs to change the 0 to a 1 since you have no 0 case above.  

Personally, I would also change the logic to be your new functionality in the if block (meaning if the cvar is > 1) because there's supposed to only be a single usable value < 2, and it makes much more sense to leave that as the else condition since it will trigger for fewer values than the if case will.  (I was taught that people expect the if block to handle more cases than the else block.)

Reply:
Both done! Agreed on if/else cases.